modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -310,25 +328,21 @@
 		timed_maturity = world.time + 1 SECONDS
 		maturity_age++
 
-		if(maturity_age == 30)
+		if(maturity_age == 20)
 			chemical_evolution++
 			to_chat(src, span_notice("You gain a chemical evolution point. Spend it to learn a new chemical!"))
-		if(maturity_age == 60)
+		if(maturity_age == 40)
 			stat_evolution++
 			to_chat(src, span_notice("You gain a stat evolution point. Spend it to become stronger!"))
-		if(maturity_age >= 90)
 			maturity_age = 0
 
 //if it doesnt have a ckey, let ghosts have it
 /mob/living/simple_animal/cortical_borer/attack_ghost(mob/dead/observer/user)
 	. = ..()
-	if(ckey)
+	if(ckey || key)
 		return
 	if(stat == DEAD)
 		return
```

Comment:
What's the point of removing the confirmation? It's possible that someone clicked on it just because they were curious _like those damn positronic brains_

Reply:
ahahahaha, fineeeee